README.md

```diff
@@ -1,17 +1,28 @@
-readers
-Source code for the e-reader apps
-
 ### About the e-reader
 
-Kontinua is an app in Swift Native, for iOS deployment. It will be used on iPads to help children get a STEM education they may not otherwise have available to them.
+Readers is an educational platform that combines a backend API with iOS and Android mobile applications. The platform delivers the Kontinua Workbook Sequence, making STEM education accessible to children who might not have access to these learning resources through traditional means.
+
+### Starting the Development Backend
+
+To run the development server that serves PDFs and metadata:
+
+1. Prerequisites:
+   - Install Docker or [Docker Desktop](https://docs.docker.com/get-started/introduction/get-docker-desktop/)
+   - Install Python 3.12
```

Comment:
Seems kinda weird once you’re already relying on Docker. You could standardize the setup by invoking the appropriate Python image.